pkgs/by-name/ex/example-robot-data/package.nix

```diff
@@ -17,8 +18,7 @@ stdenv.mkDerivation (finalAttrs: {
     owner = "Gepetto";
     repo = "example-robot-data";
     rev = "v${finalAttrs.version}";
```

Comment:
Should we take this opportunity to switch this to using `tag`? Non-blocking 😄

Reply:
Yep, thanks !